### ISSUE TITLE
Error, Unable to restore DWM

### ISSUE THREAD

**Issue by cableghost** [valinet/Win11DisableRoundedCorners#19]
As of today's latest 11 updates, it appears this approach no longer works? I receive the error "Unable to restore DWM"

**Comment by valinet** [comment]
I suggest you try [ExplorerPatcher](https://github.com/valinet/ExplorerPatcher), it already has a better version of this patch built-in and much more. Alternatively, if you just want a patch standalone, the component that patches DWM from EP is kept in a separate repository, you can follow the instructions [here](https://github.com/valinet/ep_dwm) to get that executable alone which does a much better job than this.

**Comment by ValZapod** [comment]
> As of today's latest 11 updates, it appears this approach no longer works? I receive the error "Unable to restore DWM"

No, LOL. You just updated windows and you cannot restore it anymore because it is a different original file now! LOL, what?? Just delete bak file and patch again. IT IS THAT SIMPLE, wow.

**Comment by qinlili23333** [comment]
> > As of today's latest 11 updates, it appears this approach no longer works? I receive the error "Unable to restore DWM"
> 
> No, LOL. You just updated windows and you cannot restore it anymore because it is a different original file now! LOL, what?? Just delete bak file and patch again. IT IS THAT SIMPLE, wow.

thanks it works

**Comment by hucker75** [comment]
Where is this bak file to delete?

**Comment by ValZapod** [comment]
> Where is this bak file to delete?

C:\Windows\System32\uDWM_win11drc.bak

**Comment by cableghost** [comment]
> > As of today's latest 11 updates, it appears this approach no longer works? I receive the error "Unable to restore DWM"
> 
> No, LOL. You just updated windows and you cannot restore it anymore because it is a different original file now! LOL, what?? Just delete bak file and patch again. IT IS THAT SIMPLE, wow.

I 

> DWM
Wow, people just assume. DWM does not appear to be one of the system updates, so how is one to assume it is, and how are we to know about deleting the bak file? You remind me of the computer tech guy on SNL years ago  ... dork.

**Comment by ValZapod** [comment]
> Wow, people just assume. DWM does not appear to be one of the system updates,

Of course it does. Just not every update. Just like App store dlls (has nothing to do with Store UWP app itself).

**Comment by valinet** [comment]
I don't get what the purpose of this thread is anymore, but just to mention it, the README already tells about the backup file and how it patches something belonging to the DWM, I don't understand why a Windows Update should be labelled "for DWM" to assume that it might change something regarding that... update descriptions ar enever this verbose for anything, they just generally tell what has changed and that's it.

**Comment by hucker75** [comment]
You read the readme file?!

**Comment by ValZapod** [comment]
You may add the warning that backup is not restored because the version of uDWM is newer after all you already check for that.

**Comment by valinet** [comment]
There is already a warning:

> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

Besides, I already recommended a more proper alternative that works better.

**Comment by hucker75** [comment]
What's that in English?
